src/blocks/weather.rs

```diff
@@ -15,6 +16,11 @@ use crate::util::FormatTemplate;
 use crate::widgets::button::ButtonWidget;
 use crate::widget::I3BarWidget;
 
+lazy_static! {
```

Comment:
Using lazy_static is not necessary, you can also define it as a compile time constant:
```rust
const TEXT: &str = "Test";
```
See https://doc.rust-lang.org/rust-by-example/custom_types/constants.html for more details :)